python/lsst/ts/ATWhiteLightSource/chillerEncoder.py

```diff
@@ -82,22 +82,23 @@ def _tempformatter(self, num):
             raise Exception
         if num < 0:
             sign = "-"
-        else: sign = "+" 
+        else:
+            sign = "+"
```

Comment:
I very strongly suggest replacing the entire body of this method with the following, to let the standard library do the work:
```
return f"{num*10:+05.0f}"
```
The *10 is because the number you are formatting is 10x the input value.